monitoring/ceph-mixin/dashboards/utils.libsonnet

```diff
@@ -58,8 +58,9 @@ local g = import 'grafonnet/grafana.libsonnet';
                      legend_values=legend_values),
 
 
-  addTargetSchema(expr, intervalFactor, format, legendFormat)::
+  addTargetSchema(expr, instant, intervalFactor, format, legendFormat)::
     g.prometheus.target(expr=expr,
+                        instant=instant,
```

Comment:
If you add that new argument at the end of the function, with a default value, you don't need to modify so many files. BTW, the default value should probably be `null`, not `''`. And doing the same with `intervalFactor` (90% of the time we set that to 1,instead of the default 2), we could remove a lot of unnecessary settings:
* `addTargetSchema(expr, instant, intervalFactor, format, legendFormat)` to
* `addTargetSchema(expr, legendFormat=null, format='time_series', intervalFactor=1, instant=null)`

https://github.com/grafana/grafonnet-lib/blob/6db00c292d3a1c71661fc875f90e0ec7caa538c2/grafonnet/prometheus.libsonnet#L36